core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -143,13 +146,6 @@ public void removeNetworks(String identifier) {
 
     private void removeNetwork(String networkName) {
         try {
-            try {
-                // First try to remove by name
-                dockerClient.removeNetworkCmd(networkName).exec();
```

Comment:
This seems to always fail and causes an error log. I'm just running some checks that this doesn't let any excess containers survive cleanup.